src/api/deploy-database-migration/controllers/deploy-database-migration.js

```diff
@@ -0,0 +1,41 @@
+import { deployMigrationRequestValidation } from '~/src/api/deploy-database-migration/helpers/deploy-migration-request-validation.js'
+import { getScopedUser } from '~/src/helpers/user/get-scoped-user.js'
+import { runDatabaseMigration } from '~/src/api/deploy-database-migration/helpers/run-database-migration.js'
+
+export const deployDatabaseMigration = {
```

Comment:
is it deploy or run? Be good to be consistent.

Reply:
all of the other endpoints that do similar stuff (test suite, terminal) are under deploy so it felt like it was at least consistent with whats there even if deploy is not quite the right word in this context